WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -260,7 +262,17 @@ public void Initialize(NodesCollection nodes)
 
 			Synchronizer.WhenAnyValue(x => x.UsdExchangeRate)
 				.ObserveOn(RxApp.MainThreadScheduler)
-				.Subscribe(usd => BtcPrice = $"${(long)usd}")
+				.Subscribe(usd =>
+				{
+					if (usd == default)
+					{
+						BtcPrice = LoadingText;
+					}
+					else
+					{
+						BtcPrice = $"${(long)usd}";
+					}
```

Comment:
This with ?: operator wouldn't be simplified?

Reply:
It would be shorter, not simpler. But I think we should concentrate more on functionality here. If it does what it should do and/or if there is better way to do it.

Reply:
Pls use ?: here. We are using it extensively - it is not disrupting readability.

Reply:
Ok, done.